### ISSUE TITLE
Ledger Nano S not detected @ Linux

### ISSUE THREAD

**Issue by jkn93** [harmony-one/onewallet#95]
The hardware wallet is not detected on a linux system, works fine on Windows.
I got the same issue on https://staking.harmony.one/

OS: OpenSuse Tunbleweed
Browser: Brave

**Comment by sophoah** [comment]
hey @jkn93 do you still have issue ?

**Comment by jkn93** [comment]
yes, still 'No compatible devices found' w/ both wallet and the staking website.

**Comment by sophoah** [comment]
@ethlocker is that something you can look at ?

**Comment by billytetrud** [comment]
Was this the problem back then? https://github.com/LedgerHQ/ledgerjs/issues/607

**Comment by jkn93** [comment]
Alas, no. Still can't find the device at all. I've tried it w/ the most recent chrome-version.

**Comment by billytetrud** [comment]
I'm also having problems using ledger on linux on any browser for a number of applications. My hypothesis is either a ledger firmware issue or that these applications need to update their library for interacting with ledger.

**Comment by jkn93** [comment]
Probably the latter, other wallets work fine with my ledger

**Comment by billytetrud** [comment]
@jkn93 What other wallets have you tried?

**Comment by jkn93** [comment]
not helpful for harmony, but yoroi works
